test/browser/legacy_full_page_refresh_test.rb

```diff
@@ -38,28 +54,34 @@ class LegacyPagesFullPageRefreshTest < ActionDispatch::IntegrationTest
     refute page.has_selector?("[refresh-never]")
   end
 
-  test "going to a URL that will error 500, and hitting the browser back button, we see the correct page (and not the 500)" do
+  test "going to a URL that should error 500" do
```

Comment:
At one point we had no failing tests but pages were not 500ing. You could throw the checks into another test but I prefer to separate them out.